terminal.go

```diff
@@ -246,11 +267,17 @@ func (g *LightningTerminal) Run() error {
 	// URI as soon as LND can accept the call, i.e. when the lnd sub-server
 	// is in the "Wallet Ready" state.
 	lndOverride := func(uri, manualStatus string) (bool, bool) {
```

Comment:
Nit: The comment above here for `lndOverride` is maybe a bit misplaced now that `/lnrpc.State/GetState` URI has been moved to a map. I'm fine to keep it as is if you think that the comment is still ok here :).